src/es-module-shims.js

```diff
@@ -167,6 +165,11 @@ function getOrCreateLoad (url, source) {
     s: undefined,
   };
 
+  if(url.startsWith('std:')) {
+    load.u = importShim.map.imports[url]
+    url = importShim.map.imports[url]
+  }
```

Comment:
This workaround shouldn't be needed - see https://github.com/guybedford/es-module-shims/pull/28#issuecomment-508943173 for how we should handle this.

Basically updating the resolveImportMaps function to support URL mappings, as in the SystemJS implementation, which was done in this commit - https://github.com/systemjs/systemjs/commit/a0d53c956b751c.

Reply:
Fixed 😊